### DISCUSSION TITLE
How do I restore the old feed?

### DISCUSSION THREAD

**Discussion by tprelog** [community/community#66420] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How do I turn off this new feed? I find it absolutely useless. How do I return to the old feed?

**Comment by SamJakob** [comment]
Here's a quick summary (I'll try and update this if I see more):

## My browser extension to restore the old feed
_(Pending approval on the various marketplaces)_

- **macOS Safari now supported** (see the repo link below)
- **Firefox version now available on Firefox Extension Marketplace**
- You can currently install it in Chrome from unpacked source, it's pending approval on Chrome's marketplace
  (see the repo README linked below)
- I have a version working for iOS Safari, I just need to look into distribution and whether Apple will permit it.

https://github.com/SamJakob/github-homepage

(I preferred a standalone browser extension as anyone should be able to inspect the full source of it and be confident in its safety, as opposed to analyzing a whole userscripts extension - but that's just personal preference!)

## Alternatives (Userscripts)
_(In no particular order)_

There's also this userscript here:
https://github.com/orgs/community/discussions/66260

...and someone else's here:
https://github.com/Gerrit0/old-github-feed

...and this one too:
https://github.com/wangrongding/github-old-feed
